.github/workflows/kubernetes_test.yaml

```diff
@@ -4,33 +4,34 @@ on:
   pull_request:
     paths:
       - ".github/workflows/kubernetes_test.yaml"
-      - "tests/**"
       - "tests_deployment/**"
       - "tests_e2e/**"
-      - "scripts/**"
-      - "src/**"
-      - "pyproject.toml"
-      - "pytest.ini"
+      - "src/_nebari/stages/**"
+      - "src/_nebari/template/stages/**"
```

Comment:
What do you think about running these tests for changes made to `src/**`?

Reply:
IMO, all the Python scripts under `src` should have their unit tests, and this action should only care about changes made directly to the components that interfere with the deployment itself. Though I might be overseeing something important there, do you have any code in mind that should be tested here and part of `src` ?

Reply:
I see what you're saying. My opinion is that the source code is so closely coupled with the stages/templates that are created. I'll let you make the final decision ;)